NEWS

```diff
@@ -1,5 +1,42 @@
 liblouis NEWS -- history of user-visible changes.  	-*- org -*-
 
+* Noteworthy changes in release 3.20.0
+** Braille table improvements
+- Improvements to Polish literary braille table thanks to Łukasz
+  Golonka.
+  - use correct representation of indexes for squared, cubed etc.
+  - use correct representation for fractions
+  - change symbol used for underscore from dots 46 to 6
```

Comment:
This part should probably not be in this PR, but I guess it will have to be resolved when merging

Reply:
I've rebased.